### PR TITLE
Cult stun nerf + Zealots gives SecHuds

### PR DESCRIPTION
**Pull request by JohnFulpWillard**
## About The Pull Request

This nerfs the cult's stun ability, causing mindshielded people to drop held items, jitter/stutter, bleed 5% of their blood and take 60 stamina damage (which means 2 consecutive cult stuns will result in stamcrit), while keeping the actual stun for non mindshielded people.
To slightly balance this out we've made a Fulp-only version of the Zealot's blindfold which has Security huds so they can tell who is mindshielded.

## Why It's Good For The Game

The cult stun should be for early game converting and not for winning fights against armed Security Officers lategame.
Also, unless the cult's target is the Chaplain, all it takes is one click to take your target out. This way, even if their target isn't the Chaplain, it still won't just take one click to win.
/tg/ justified that this nerf was bad because "Cult doesn't have security huds", so the blindfold is my compromise
They can also still put you into stamcrit if they use it twice, but that means they're missing another spell they could've had, which at least makes it slightly more balanced.

## Changelog

🆑
balance: Cult stun causes mindshielded people to jitter rather than stun
balance: Zealot's blindfold gives SecHUDs
/🆑

### CODE REVIEW

File: code/modules/antagonists/cult/blood_magic.dm [hunk 1]
```diff
@@ -452,6 +452,8 @@
 				C.stuttering += 15
 				C.cultslurring += 15
 				C.Jitter(1.5 SECONDS)
+				C.silent += 6
+				C.stuttering += 15
```
Comment (ATH1909) [inline code/modules/antagonists/cult/blood_magic.dm]: ???

**Comment by ATH1909** [comment]
I'd recommend making the influencer's blindfold give you both secHUDs AND medHUDs, then just replacing the zealot blindfold's spot in the archives with the influencer's blindfold.

**Comment by EgorDinamit** [comment]
"Powercrepe", but yeah, you would be better off with simply making it use both HUDs.

**Comment by JohnFulpWillard** [comment]
Gonna remake this without the 70 commits

**Pull request closed without merging**